src/frontend/komponenter/Fagsak/Vilkårsvurdering/Vilkårsvurdering.tsx

```diff
@@ -39,6 +40,7 @@ const HentetLabelOgKnappDiv = styled.div`
     .knapp__spinner {
         margin: 0 !important;
     }
+    margin-bottom: ${NavdsSpacing2};
```

Comment:
Dette er bunnmargin for "Registeropplysninger hentet [...]" under sidetittelen:
<img width="641" alt="image" src="https://user-images.githubusercontent.com/2379098/184860753-e67ba30f-4559-4065-9a1f-9fe65b702aa7.png">